release-notes/release-notes-2-2.md

```diff
@@ -1,38 +1,22 @@
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
 <!-- Copyright Contributors to the ODPi Egeria project. -->
 
-# Release 2.2 (Planned August 2020)
+# Release 2.2 (August 2020)
 
-Release 2.2 focuses on providing the secured self-service data access point
-for a data lake.  There are two main features:
+Below are the highlights of this release:
 
-  * **Metadata Controlled Security** where security tags based on the metadata attachments to the assets
-    drive the enforcement engine managing access control
-  * **Data Virtualization Configuration** where the data virtualization engine that is providing
-    the endpoint for the self-service access point.
+ * Additional connectors are now placed in our assembly under server/lib without dependencies. If you need to use a connector that requires additional dependencies that are not alreadypart of the server chassis, you will need to add those libraries here.
+ * The docker image has been updated to use a later openjdk alpine base image due to incompatabilities with our Janus support in the old images for java 8.
+ * In the VDC chart, 2 new values have been added, 'ibmigc.connectorversion' and 'atlas.connectorversion'. In this release these are set to use the 2.1 connectors, since the connectors run to a different release cycle than the main egeria code. Once new connectors are released you can update these values to get the latest connectors 
+ * Further code to support lineage has been added, but in this release it remains in development and is not yet ready for use.
+ * User interface improvements.
+ * Ongoing bug fixes and refactoring especially in subject-area omas.
 
-Below are the highlights:
+Known Issues
+ * In the VDC helm chart, the Atlas initialization job fails to complete. This is due to a problem with the Atlas server and SOLR. See https://github.com/odpi/egeria/issues/3587 for more information.
```

Comment:
We should say Apache Atlas and Apache Solr.

Reply:
fixed